### PR TITLE
Add instructions for vim-plug

### PR DESCRIPTION
**Pull request by dkarter**
Although vim plug would normally be simple enough and somewhat similar to vundle, it is important to give it the `as` option to make it install dracula in it's own 'dracula' directory. This is because the repo is abnormally named `vim` (that's not a standard in the community) and can cause conflicts if others do the same.

### CODE REVIEW

File: src/content/vim/index.html [hunk 1]
```diff
@@ -21,6 +21,10 @@
     <pre><code>Plugin 'dracula/vim'
 :PluginInstall</code></pre>
 
+	<p>If you <a href="https://github.com/junegunn/vim-plug">use vim-plug</a> (`as` will install the plugin in a directory called 'dracula' instead of just 'vim'):</p>
+    <pre><code>Plug 'dracula/vim', { as: 'dracula' }
```
Comment (bengadbois) [inline src/content/vim/index.html]: This should be `{ 'as': 'dracula' }` with quotes around the 'as'

Comment (dkarter) [inline src/content/vim/index.html]: fixed.

**Comment by zenorocha** [comment]
Hey guys, sorry for taking so long to review this.

@BlakeWilliams can you review and let me know if this is good to merge?

Thanks!

**Comment by BlakeWilliams** [comment]
Yep, this works and looks good to me. :shipit:

**Comment by zenorocha** [comment]
Thanks guys!

**Pull request merged**